fixtures/docker.yaml

```diff
@@ -0,0 +1,17 @@
+command: docker
```

Comment:
Same for the YAML; do we expect this to include the `--help` flag (or at least custom-defined flags?)